CallCenterSample/Helpers/TextAnalyticsHelper.cs

```diff
@@ -163,28 +170,19 @@ public static async Task<KeyPhrasesResult> GetKeyPhrasesAsync(string input, stri
 
             if (!string.IsNullOrEmpty(input))
             {
-                KeyPhraseBatchResult result = await AnalyticsClient.KeyPhrasesAsync(new MultiLanguageBatchInput(
-                    new List<MultiLanguageInput>()
-                    {
-                        new MultiLanguageInput(language, "0", input)
-                    }));
+                ExtractKeyPhrasesResultCollection result = await AnalyticsClient.ExtractKeyPhrasesBatchAsync(
```

Comment:
same comment here. See sample for [KeyPhrases](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample3_ExtractKeyPhrases.md#extracting-key-phrases-from-a-single-document)

Reply:
https://github.com/Azure-Samples/cognitive-services-csharp-call-center/pull/2#discussion_r528560302